palantir-java-format/src/test/resources/com/palantir/javaformat/java/testdata/palantir-lamda-cast.output

```diff
@@ -10,7 +10,7 @@ public class Foo {
     }
 
     static String baz(Optional<String> optional) {
-        return optional.orElseGet(
-                (String) () -> bar(Optional.of("some thing that is very very very looooooong")).get());
+        return optional.orElseGet((String) () ->
+                bar(Optional.of("some thing that is very very very looooooong")).get());
```

Comment:
This changed beause it doesn't allow the dot in the `bar(...).get()` expression to exceed 80 chars anymore. It now ends up at exactly 80 chars, and to prove that, if I add another character to the string, then the `.get()` gets pushed onto the next line, as in:

```java
return optional.orElseGet((String) () -> bar(Optional.of("some thing that is very very very loooooooong"))
        .get());
```